frontend/public/components/monitoring/receiver-forms/routing-labels-editor.tsx

```diff
@@ -72,129 +48,52 @@ export const RoutingLabelEditor = ({ formValues, dispatchFormChange, isDefaultRe
 
   const { t } = useTranslation();
 
-  const InvalidLabelName = () => (
-    <span data-test-id="invalid-label-name-error">
-      {t('public~Invalid name.')}
-      <Tooltip
-        content={
-          <p>
-            {t(
-              "public~Label name must not begin with a digit and contain only alphanumeric characters or '_'.",
-            )}
-          </p>
-        }
-      >
-        <InfoCircleIcon className="co-icon-space-l" />
-      </Tooltip>
-    </span>
-  );
-
   return (
     <div data-test-id="receiver-routing-labels-editor" className="form-group">
       <SectionHeading text={t('public~Routing labels')} required={!isDefaultReceiver} />
-      <p className="co-help-text">
+      <p className="co-help-text pf-u-mb-md">
         <Trans ns="public">
-          Firing alerts with labels that match all of these selectors will be sent to this receiver.
-          Label values can be matched exactly or with a{' '}
+          Firing alerts with labels that match all of these{' '}
           <ExternalLink
-            href="https://github.com/google/re2/wiki/Syntax"
-            text={t('public~regular expression')}
-          />
-          .
+            href="https://github.com/prometheus/alertmanager/blob/e3fb99cc2d24330fdb81aa1d466c19b3de328b5b/docs/configuration.md#matcher"
```

Comment:
Lets rather link to the alerting configuration docs
```suggestion
            href="https://prometheus.io/docs/alerting/latest/configuration/#matcher"
```

Reply:
Good call.  Updated.